### PR TITLE
Use React Router (Redo): Pt 1/3: Add Routes, Copy /pages/* into /real-pages

### PR DESCRIPTION
**Pull request by jessie-ross**
**NOTE: We should merges this series of PRs together.**

**This PR is the first of two migrating this app from NextJS to CRA, this one changes our underlying routing to use react router.**

- Move content in `/src/pages` into `/src/pages-real`, the **content of these files in unchanged.**
- Create new routing system at `/src/Routes.tsx`
- Makes `/pages/index.tsx` run `/src/Routes.tsx`
- Symbolically link all files in `/pages` to `/pages/index.tsx`

Part of a series on #212 

## How has this been tested?

- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
- Manual testing with mock data (locally or on Vercel)

### CODE REVIEW

File: src/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import Metrics from 'src/pages-real/Metrics'
+import Tags from 'src/pages-real/Tags'
+
+/**
+ * Let's keep routing simple and minimal.
+ * - Do not use dynamic or nested routing!
+ * - Get all your route information at the top level.
+ * - Try not to delete or change an existing route: comment a route as deprecated and redirect.
+ */
+function Routes(): JSX.Element {
```
Comment (yanirs) [inline src/Routes.tsx:17]: @jessie-ross shouldn't there be routes for the auth callback? Not sure if it's related, but running with `NEXT_PUBLIC_PRODUCTION_CONFIG_IN_DEVELOPMENT=true npm run dev`, I get stuck in a redirect loop.

Comment (jessie-ross) [reply src/Routes.tsx:17]: Ah oops, yep there should be, must have accidentally deleted it.

Comment (jessie-ross) [reply src/Routes.tsx:18]: This should now be resolved :)

Comment (jessie-ross) [reply src/Routes.tsx:18]: Nope actually

Comment (jessie-ross) [reply src/Routes.tsx:18]: Ok, now it should be 🤞

**Review by yanirs — APPROVED** [review]
LGTM with the stacked changes :slightly_smiling_face:

**Pull request merged**